values/roundtriptest/int_test.go

```diff
@@ -13,19 +13,27 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestPositiveIntDictionaryEncodeAndDecode(t *testing.T) {
+const (
+	mockDataOutlierMax = 1 << 32
+	mockDataOutlierMin = -(1 << 32)
+)
+
+func TestPositiveIntRawSizeEncodeAndDecode(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
 	meta := values.IntValuesMetadata{
 		Size: 512,
-		Min:  1,
-		Max:  1,
+		Min:  0,
+		Max:  mockDataOutlierMax,
```

Comment:
Is this used to determine whether we are using varint encoding?

Reply:
Yes, I'll add a comment above the const defs.